deploy.sh

```diff
@@ -20,6 +20,7 @@ function deployRevision() {
         --application-name "$INPUT_CODEDEPLOY_NAME" \
         --deployment-group-name "$INPUT_CODEDEPLOY_GROUP" \
         --description "$GITHUB_REF - $GITHUB_SHA" \
+        --file-exists-behavior "${INPUT_CODEDEPLOY_FILE_EXISTS_BEHAVIOR}" \
```

Comment:
Minor nit that our other variables here are just plain `$VAR` and this one became `${VAR}`.

Reply:
I was going to use bash parameter substitution syntax: `${INPUT_CODEDEPLOY_FILE_EXISTS_BEHAVIOUR:-DISALLOW}` but realised we can set a default through the `actions.yaml` - will amend this.